app/src/main/res/values/strings.xml

```diff
@@ -173,4 +173,6 @@
     <string name="feedback_invalid_email">Input valid email address</string>
     <string name="drawer_subheader_games">Games</string>
     <string name="fork_me">Fork me</string>
+    <string name="arrowTagImage">Arrow Tagging Game</string>

```

Comment:
No we use it in couple of places. It helps the translator.

On Tue, Aug 4, 2015, 6:20 AM Michael Prentice <notifications@github.com>
wrote:

> In app/src/main/res/values/strings.xml
> <https://github.com/gdg-x/frisbee/pull/455#discussion_r36154557>:
>
> > @@ -173,4 +173,6 @@
> >      <string name="feedback_invalid_email">Input valid email address</string>
> >      <string name="drawer_subheader_games">Games</string>
> >      <string name="fork_me">Fork me</string>
> > +    <string name="arrowTagImage">Arrow Tagging Game</string>
>
> Never heard of something like that. I can add it though. I don't see any
> other strings with the cd_ prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/455/files#r36154557>.
>